examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -50,6 +50,10 @@ function example_API_ConfigurationInRegionDefault() {
   Configurations.InRegion.Default.v1();
 }
 
+function example_API_ConfigurationInRegionDefaultLatest() {
+  Configurations.InRegion.Default.latest();
```

Comment:
these warnings above are indicating that this function is never called.  at the bottom of this file you'll see a `main` function that calls all of the other functions.  this is how we can make sure the code runs properly when our github CI jobs are executed.